### DISCUSSION TITLE
Coming and Going

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3678] [Team Posts]
Just a thought to share with the team.  Most of us are contributing to this project on a voluntary basis and we have other projects that call for our attention (or life happens). If that's the case and you have any Pull Requests assigned to you that releases are dependent on, please let people know so that we can not waste time asking if you're going to finish it and someone else can take it over.  Ideal would be to finish the PR before taking the hiatus, but that can't always be helped.

**Comment by SteveIreland** [comment]
Are you talking to me?     Are you talking to ME?!?!?

**Comment by SteveIreland** [comment]
Actually I just checked and my remaining pull request is in the questionable category and isat the earliest scheduled for 3.4

**Comment by objecttothis** [comment]
> Are you talking to me? Are you talking to ME?!?!?

I don't feel lucky, so no ;).  Actually, I wasn't aiming at any one of us, but wanted to propose the team norm to help reduce slowdown of the "are you there? Can you finish this?" posts so someone can already know that whatever dev had to bow out and someone else can take it.

**Comment by jekkos** [comment]
Ok indeed, coming months will be rather busy for me (picking up social duties after lockdown ended here in Belgium), so I'll be chiming in and out from time to time. But good point I think it makes sense to try and keep the team updated if one is going to be off for a longer time.;

**Comment by objecttothis** [comment]
Thanks for the heads up.  Do you think we can get 3.3.2 out before you bow out?

**Comment by jekkos** [comment]
Let's try. If the system info thingy is sorted I gues we should be good?

**Comment by WebShells** [comment]
Maybe @objecttothis was talking about me, having some life issues. Will try to get back the soonest!
Sorry for the delays team, stay safe!

**Comment by objecttothis** [comment]
> Maybe @objecttothis was talking about me, having some life issues. Will try to get back the soonest!
> Sorry for the delays team, stay safe!

Sorry to hear about your troubles @WebShells.  I wasn't trying to single anyone out.
